odk2-src/advanced-topics-architect.rst

```diff
@@ -0,0 +1,15 @@
+Deployment Architect Advanced Topics
+=======================================
+
```

Comment:
I would repeat your definition of a Deployment Architect here is probably wise since a reader may not have read the intro